app/assets/stylesheets/style/copyright.scss

```diff
@@ -1,9 +1,27 @@
 .copyright {
   background: $footer-gray;
-  border-top: solid 1px $gray-headline;
   font-size: 12px;
   padding: 11px 0 7px;
 
+  .year {
+    float: left;
+  }
+
+  .contact {
+    float: right;
+  }
+
+  @media (max-width: 500px) {
+    .contact,
+    .year {
+      float: none;
+      margin-left: auto;
+      margin-right: auto;
+      text-align: center;
```

Comment:
Properties should be ordered float, margin-left, margin-right, padding, text-align